contributing.md

```diff
@@ -0,0 +1,37 @@
+# What this repository is
+This repository contains sources of the Introduction to Python course.
+
+# Contributing
+We love contributions!
+We are happy to see fixes of the existing bugs and tasks and tests mistakes.
+The current tasks can be found in the [open issues](https://github.com/jetbrains-academy/introduction_to_python/issues) in the project.
+If you have some questions or find bugs or mistakes, please do not hesitate to open new ones.
+
+Please, add a comment to the issue, if you're starting work on it.
+
+If you add some common functionality, e.g. for the test system, it is important to add comments to describe new behaviour.
+This will help other developers and users to use them correctly.
+
+## Submitting patches
+The best way to submit a patch is to [fork the project on GitHub](https://help.github.com/articles/fork-a-repo/)
+and then send us a [pull request](https://help.github.com/articles/creating-a-pull-request/)
+to the `master` branch via [GitHub Pull requests](https://github.com/jetbrains-academy/introduction_to_python/pulls).
+
+If you create your own fork, it might help to enable rebase by default
```

Comment:
If you create your own fork, it might be helpful to enable rebase by default when you pull.